scenario_player/tasks/blockchain.py

```diff
@@ -86,6 +89,14 @@ def query_blockchain_events(
     ]
 
 
+def verify_config(config, required_keys):
+    all_required_options_provided = all(key in config.keys() for key in required_keys)
+    if not all_required_options_provided:
```

Comment:
Nitpick: I think it'd be shorter/nicer to just do
```python
    if not all(key in config for key in required_keys):
    ...
```

Reply:
You can shorten this by stopping in the first key you don't find:

```python
if any(key not in config for key in required_keys):
  raise ScenarioError(
            "Not all required keys provided. Required: " + ", ".join(required_keys)
        )
```